sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/EventDataAssert.java

```diff
@@ -82,4 +84,34 @@ public EventDataAssert hasAttributesSatisfying(Consumer<Attributes> attributes)
     assertThat(actual.getAttributes()).as("attributes").satisfies(attributes);
     return this;
   }
+
+  /** Asserts the exemplar has filtered attributes matching all {@code assertions}. */
```

Comment:
```suggestion
  /** Asserts the event has attributes matching all {@code assertions}. */
```
Same for other new methods in this class